sources/models/src/modeled_types/shared.rs

```diff
@@ -240,6 +240,9 @@ mod test_url {
             "http://localhost",
             "localhost/path",
             "localhost",
+            "localhost:8080",
+            ".internal",
+            ".cluster.local"
```

Comment:
Should these last two really be good URLs?

Reply:
Since the `no_proxy` settings is passed as a list of URLs (the modelled type), I want to allow users to specify `no_proxy` for everything under a domain name suffix and apparently any string without special characters can be a valid URL if you prefix it with a scheme.